src/amo/components/Footer/index.js

```diff
@@ -22,60 +24,145 @@ export class FooterBase extends React.Component {
 
     return (
       <footer className="Footer">
-        <div className="Footer-language-and-links">
-          <LanguagePicker
-            location={location}
-            ref={(ref) => { this.languagePicker = ref; }}
-          />
-          <ul className="Footer-links">
-            <li>
-              <a
-                href="https://www.mozilla.org/privacy/websites/"
-                className="Footer-link Footer-privacy"
-              >
-                {i18n.gettext('Privacy policy')}
-              </a>
-            </li>
-            <li>
-              <a
-                href="https://www.mozilla.org/about/legal/"
-                className="Footer-link Footer-legal"
-              >
-                {i18n.gettext('Legal notices')}
-              </a>
-            </li>
-            <li>
-              <a
-                href="https://developer.mozilla.org/Add-ons/AMO/Policy/Contact"
-                className="Footer-link Footer-fileissue"
-              >
-                {i18n.gettext('Report a bug')}
-              </a>
-            </li>
-            <li>
-              <a
-                href="#desktop"
-                className="Footer-link Footer-desktop"
-                onClick={handleViewDesktop}
-                ref={(ref) => { this.desktopLink = ref; }}
-              >
-                {i18n.gettext('View classic desktop site')}
-              </a>
-            </li>
+        <div className="Footer-wrapper">
+          <a
+            className="Footer-mozilla-link"
+            href="https://mozilla.org/"
+            title={homepageText}
+          >
+            <Icon
+              alt={homepageText}
+              className="Footer-mozilla-logo"
+              name="mozilla"
+            />
+          </a>
+
+          <section className="Footer-amo-links">
+            <h4 className="Footer-links-header"><Link href="/">{i18n.gettext('Add-ons')}</Link></h4>
+            <ul className="Footer-links">
+              <li>
+                <Link
+                  href="/about"
+                  prependClientApp={false}
+                >
+                  {i18n.gettext('About')}
+                </Link>
+              </li>
+              <li>
+                <a href="http://blog.mozilla.com/addons">{i18n.gettext('Blog')}</a>
```

Comment:
HTTPS

Reply:
Heh nice catch -  I copy + pasted this from the legacy frontend.